src/webgpu/api/validation/render_pass/render_pass_descriptor.spec.ts

```diff
@@ -713,3 +713,26 @@ g.test('multisample_render_target_formats_support_resolve')
       colorAttachments: [colorAttachment],
     });
   });
+
+g.test('occlusion_query_set_type')
+  .desc('Test that occlusion query set only works if the type is occlusion')
+  .paramsSimple([
+    { querySetState: 'valid' as const, queryType: 'occlusion', _success: true },
+    { querySetState: 'invalid' as const, queryType: 'timestamp', _success: false },
```

Comment:
This changes two things which will both separately make the render pass encoding fail. I think there is already a test for invalid query set which I found searching for `createQuerySetWithState`, see the test `occlusion_query,invalid_query_set`.

Actually, it looks like there is already `occlusion_query,query_type` as well, so please check if that's testing the same as your new test.

Sorry that this is poorly organized. It's unfortunate that we have two different places for this.

Reply:
Ah, my bad to miss checking 'occlusion_query,query_type' test in advance. It's likely this test is duplicated to the test. So, it's likely it's redundant to add. Let me abandon this PR and try to check if there is a test before adding a new test.